src/services/GeojsonService.js

```diff
@@ -25,13 +27,49 @@ export default class GeojsonMapService {
     }
   }
 
-  static getHTMLWikipediaLink(link, popupText) {
-    if (link !== '') {
-      return `<p class=""><a  class="btn btn-light" target="_blank" href='${link}'><i class="fab fa-wikipedia-w"></i></a></p>`;
+  static getHTMLWikipediaLink(link, popupText, data) {
```

Comment:
WDYT about renaming `data` to something a bit more auto explicative like `wikidataDetails`, `dataDetails`, or similar?